functests/5_latency_testing/latency_testing.go

```diff
@@ -160,6 +164,41 @@ var _ = Describe("Run tests of latency measurement tools with different values o
 			table.Entry("[test_id:42856] Hwlatdetect - Verify that the latency tool test should print an expected error message when passing invalid environment variables values", getNegativeTests(hwlatdetect), negativeTesting),
 		)
 	})
+
+	Context("Verify tests output with updated performance profile", func() {
+		It("[test_id:52388] should skip oslat test when isolated CPUs amount is less than 2", func() {
+			profile, err := profiles.GetByNodeLabels(testutils.NodeSelectorLabels)
+			Expect(err).ToNot(HaveOccurred())
+			initialIsolated := profile.Spec.CPU.Isolated
+			initialReserved := profile.Spec.CPU.Reserved
+			//updated both sets to ensure there is no overlap
+			latencyIsolatedSet := performancev2.CPUSet("1")
+			latencyReservedSet := performancev2.CPUSet("0")
+			profilesupdate.UpdateIsolatedReservedCpus(latencyIsolatedSet, latencyReservedSet)
```

Comment:
I'm very wary of tests which change their environment. This is intrisincally more fragile and slow. Can we maybe start _checking_ the cluster is not properly configuring and skipping/failing if not the case?

Reply:
if we don't change the environment this test will always be skipped.

Reply:
ok, that's fair. But do we need to change the environment _within_ the test? any alternatives?

Reply:
because oslat requires currently for 4.10 at least  >=2, that leaves us with 1 for this test, but setting 1 will be skipped because of the SMTalignment.
@Tal-or are we planning to change oslat minimum cpus to 3 in 4.10? don't we have bug for that? 
and what about  4.9 do we expect any changes?

Reply:
> @Tal-or are we planning to change oslat minimum cpus to 3 in 4.10? don't we have bug for that?

We do. once the fix is in I'll backport to 4.10

> and what about 4.9 do we expect any changes?

Nothing that I'm aware of

Reply:
thanks for the info Talor, @fromanirh shall we keep this new test or wait for the bug fix?
(if we keep it and re-adjust it later it means replacing the whole [new context](https://github.com/openshift-kni/performance-addon-operators/pull/904/commits/04715f26f102aa4669692148a9347a18cea63c51#diff-430c986deb22f7ffd2962b8759ac1068ad8d55477c7588d04163522a33d85af0R168) with one line like other [testcases](https://github.com/openshift-kni/performance-addon-operators/pull/904/commits/04715f26f102aa4669692148a9347a18cea63c51#diff-430c986deb22f7ffd2962b8759ac1068ad8d55477c7588d04163522a33d85af0R276), but OTOH if we don't merge it we don't have a test covering this scenario (which is a minor case) until the new fix lands.

Reply:
it seems the lesser evil is to merge this test as-is and re-evaluate later